base/shell.jl

```diff
@@ -253,3 +253,62 @@ julia> Base.shell_escape_posixly("echo", "this", "&&", "that")
 """
 shell_escape_posixly(args::AbstractString...) =
     sprint(print_shell_escaped_posixly, args...)
+
+
+function print_shell_escaped_winsomely(io::IO, args::AbstractString...)
```

Comment:
Should we add a `force`  option with `false` default similar to  the code in https://blogs.msdn.microsoft.com/twistylittlepassagesallalike/2011/04/23/everyone-quotes-command-line-arguments-the-wrong-way/ 

In which case when set to true will always add quotes, even if not needed?

Reply:
https://blogs.msdn.microsoft.com/twistylittlepassagesallalike/2011/04/23/everyone-quotes-command-line-arguments-the-wrong-way/  their argument  quoting criteron also checks for `\n` and `\v`, I see no mention of these in the docs so not sure why they add them?

Reply:
My test show that at least `\n` doesn't split arguments, so we can ignore `\n` and `\v` . Adding `force` is probably not needed, since I can't imagine it being good for anything, anyways.